### ISSUE TITLE
[BUG] App does not start anymore

### ISSUE THREAD

**Issue by azomDev** [Kin69/EasyNotes#221]
## Required Information

- [x] I’m using the latest EasyNotes version
- [x] The device is not rooted, and nothing has been removed via ADB

Note that I am using grapheneOS, but it should not interfere afaik.

### Steps to Reproduce the Bug
Not sure what it is for the moment since I am unable to start the app to replicate the bug (and I don't want to uninstall the app until I find where the notes are stored to back then up).

However, I was just starting to use the app and after my first note, I tried going into the settings and enabled the locked storage or whatever it is, then activated fingerprint because I initially thought it was for the locked storage, than I think I restarted the app and then deactivated fingerprint and finally deactivated locked storage (I am not sure of the last step tho).

I did play with some other settings like background color to extreme battery amoled thing and other settings but nothing that I can remember precisely or that are "weird" settings.

### Expected Behavior
the app should launch, it however does not really. When I launch the app, I just see the screenshot I put below:

### Screenshots
Edit: Ignore the square notification, it's just an app I am writing so I was testing the notifications.
![Screenshot_20241227-145609.png](https://github.com/user-attachments/assets/c9672e28-ddca-4e6d-aef2-6a6a171f0e78)

### Smartphone Information
- **Device:** Google Pixel 8
- **OS:** grapheneOS 2024121200
- **EasyNotes Version:** 1.4 (9)
For the EasyNote version, it is what I saw on my device and I am unsure if (9) is part of the version. If not you can ignore it. I just downloaded it about 30 minutes ago on December the 27th 2024 using F-droid

### Steps I tried to fix the issue (in order)
- tried to close completely the app (swipe it) and start it again
- tried to clear cache
- tried to restart the phone

**Comment by Kin69** [comment]
@azomDev Can you try pressing back button when on that screen if that doesn't work join discord server and i will provide tutorial how to restore notes i will try fixing that bug as soon as possible but from what i know it is not something unfixable so your notes should be safe.

**Comment by azomDev** [comment]
@Kin69 for the quick response!

Triggering the back button with either ways (I tried to switch to the 3 buttons on the button and it also did not work). In both cases, it puts my on my home screen.

As for my "notes", I actually just have one right now since I just installed the app, so nothing that important. I am probably able anyways to get it back since I have a decent understanding of Android and app development. (iirc, it's going to be in like /data/com.kin.easynotes or something similar, that's not really a problem, it's more about the path after that one I would need to figure out)

However if you think it would be easier to go on discord to try fixing the issue I have an account so I can get on right away :)
